lib/migrate/Migrator.js

```diff
@@ -108,16 +108,24 @@ class Migrator {
         return value;
       })
       .then(([all, completed]) => {
-        const migrationToRun = getNewMigrations(
+        const newMigrations = getNewMigrations(
           this.config.migrationSource,
           all,
           completed
-        ).slice(0, 1);
+        );
+        const migrationToRun = name
```

Comment:
Can you also throw an error if name was specified but migration was not found?

Reply:
done, throws an error in `up` and in `down` methods